Copyright (c) 2000-2003 Intel Corporation 
All rights reserved. 

Redistribution and use in source and binary forms, with or without 
modification, are permitted provided that the following conditions are met: 

* Redistributions of source code must retain the above copyright notice, 
  this list of conditions and the following disclaimer. 
* Redistributions in binary form must reproduce the above copyright notice, 
  this list of conditions and the following disclaimer in the documentation 
  and/or other materials provided with the distribution. 
* Neither name of Intel Corporation nor the names of its contributors 
  may be used to endorse or promote products derived from this software 
  without specific prior written permission.
 
THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS 
``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT 
LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR 
A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL INTEL OR 
CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, 
EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, 
PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR 
PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY 
OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS 
SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.

Date: Feb 5, 2016 9:37 AM

From: 560bf904ed7c707bd45e474d825e80d71638fcb4 Feb 5, 2016 12:54 PM

Subject: [PATCH] RDKSYSINTSW-1306: Fix out of tree build for libupnp

	This is windows specific code. It assumes that
	build is happening in same dir as sources. This
	breaks out of tree builds.

	We dont have windows case, so just delete it
	
Source: COMCAST

Upstream-Status: Pending

Signed-off-by: Khem Raj <khem_raj@cable.comcast.com>
---


Index: libupnp-1.6.19/configure.ac
===================================================================
--- libupnp-1.6.19.orig/configure.ac
+++ libupnp-1.6.19/configure.ac
@@ -710,12 +710,3 @@ AC_CONFIG_FILES([
 
 
 AC_OUTPUT
-
-#
-# Files copied for windows compilation.
-#
-echo "configure: copying \"autoconfig.h\"          to \"build/inc/autoconfig.h\""
-cp autoconfig.h build/inc/autoconfig.h
-echo "configure: copying \"upnp/inc/upnpconfig.h\" to \"build/inc/upnpconfig.h\""
-cp upnp/inc/upnpconfig.h build/inc/upnpconfig.h
-
